easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-2023a.yml

```diff
@@ -16,3 +16,7 @@ easyconfigs:
   - MAFFT-7.520-GCC-12.3.0-with-extensions.eb
   - ncbi-vdb-3.0.10-gompi-2023a.eb
   - MetalWalls-21.06.1-foss-2023a.eb
+  - CP2K-2023.1-foss-2023a.eb:
+      # see https://github.com/easybuilders/easybuild-easyconfigs/pull/20951
+      options:
+        from-pr: 20951
```

Comment:
```suggestion
  - CP2K-2023.1-foss-2023a.eb:
      options:
        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/20951
        from-commit: a92667fe32396bbd4106243658625f7ff2adcd68
```

Reply:
@laraPPr is on holidays so I will commit this suggestion